src/Fixture/FixtureCreator.php

```diff
@@ -409,7 +409,7 @@ private function createComposerProject(): void {
   private function getProjectTemplateString(): string {
     switch (TRUE) {
       case $this->isProjectTemplateSut():
-        return "{$this->projectTemplate}:@dev";
+        return "{$this->projectTemplate}:dev-bologna";
```

Comment:
This is necessary to convince Composer to use the path repo instead of remote package version. It seems like ORCA doesn't work its sut-branch-to-version-number magic on the project template like it does on other SUTs (i.e. requiring version 12.9999.9999.999 or whatever for BLT). If it did that, maybe we wouldn't need this hack.